components/webui/imports/ui/SearchView/SearchView.jsx

```diff
@@ -212,15 +233,15 @@ const SearchView = () => {
         </div>
 
         {showSearchResults && <SearchResults
+            fieldToSortBy={fieldToSortBy}
```

Comment:
It might still be intuitive to still group `var` with `setVar`. What do you think?

Reply:
Cognitively it might be harder to find property names when the props are not strictly alphabetically sorted. It could be easier if we can put the `var` and `setVar` on the same line. e.g.,
```jsx
<SearchResults
    fieldToSortBy={fieldToSortBy} setFieldToSortBy={setFieldToSortBy}/>
```

I can't find a proper way to configure ESLint to automatically achieve that though. Also that will violate the rule to have only one prop per line. 

I am not really sure if we wish to turn off rule [`react/jsx-sort-props`](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-sort-props.md). On the other hand, all the setters are grouped together and alphabetically sorted because they start with "set...", which helps finding a state setter in the group in my opinion.

Reply:
Ah, if it's a done automatically by the linter, let's leave it.